tools/add_triage_signature.py

```diff
@@ -2149,6 +2098,44 @@ def _process_ticket(self, url, issue_key):
             self._update_triaging_ticket("\n".join(host_messages))
 
 
+class UserManagedNetworkingLoadBalancer(ErrorSignature):
+    lb_operators = set(['authentication', 'console', 'ingress'])
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(
+            *args,
+            **kwargs,
+            function_impact_label="umh_load_balancer_unhealthy",
+            comment_identifying_string="h1. Probably user managed load-balancer issues, could theoretically be prevented with MGMT-11941",
+        )
+
+    def _process_ticket(self, url, issue_key):
+        md = get_metadata_json(url)
+        cluster_md = md["cluster"]
+        if cluster_md["user_managed_networking"] is not True and cluster_md["user_managed_networking"] != "true":
```

Comment:
```suggestion
        if cluster_md.get("user_managed_networking", False):
```

Reply:
The value is a simple boolean, there's no reason to believe it will contain the string "false", did you see anything like that?

Reply:
I just copied the check from https://github.com/openshift-assisted/assisted-installer-deployment/blob/e7b869a3cff31dde659ee0d6fada0aca2c226289/tools/add_triage_signature.py#L672 , I dont know the reason to check both types ... maybe it's for backwards compatibility and it's not neede anymore :?

@mkowalski do you know about this?

Reply:
Yeah, back then when I wrote this I tried to find an ultimate answer to “what is the type here” question, but given it’s Python without typing, I didn’t manage to. If anyone is perfectly sure we have here one type and not the other I’m fine with this

Reply:
Okay then let's go with my edit suggestion